src/YAML.jl

```diff
@@ -131,6 +131,9 @@ function iterate(it::YAMLDocIterator, _ = nothing)
     return doc, nothing
 end
 
+Base.IteratorSize(::Type{YAMLDocIterator}) = Base.SizeUnknown()
```

Comment:
This was implemented (recently) on line 119 but here it looks more correct, or at least more consistent with methods from Base. Can you remove the line 119 method (and probably the comment, unless you think it's worth keeping)?

Reply:
Good catch, done.